### PR TITLE
doc: add CTC meeting minutes for 2016-11-02

### PR DESCRIPTION
**Pull request by targos**
Copied from https://github.com/nodejs/node/pull/9430 and fixed nits noticed by @Trott.

### CODE REVIEW

File: meetings/2016-11-02.md [hunk 1]
```diff
@@ -0,0 +1,476 @@
...
+
+Weve been better at doing things in GitHub, but havent had much usage of the
+CTC review process.
+
+Id like to give it another week. If we continue to not use it thats probably a
+sign we dont need it.
+
+@Fishrock123: I havent seen this used yet.
+
+@trott: That seems to support my conclusion that we cant conclude yet.
```
Comment (Trott) [inline meetings/2016-11-02.md]: Nit: cant -> can't

File: meetings/2016-11-02.md [hunk 2]
```diff
@@ -0,0 +1,476 @@
...
+
+@trott: Judging from the tracker, Brian is the strongest voice opposed. Is there
+a way we can get to consensus here?
+
+@trevnorris: @bmeck do you know what part of the spec caused this change?
+
+Are we painting ourselves in the corner, since the spec change means its going
+to have to change at some point?
+
+@trott: Principal of dont break stuff on purpose.
```
Comment (Trott) [inline meetings/2016-11-02.md]: Nit: Principal -> Principle

Comment (Trott) [reply meetings/2016-11-02.md]: Although, honestly, not sure what I said there, but it doesn't look like it was adding much to the conversation. The line can be removed.

**Review by Trott — APPROVED** [review]
Some nits, but LGTM

**Review by mhdawson — APPROVED** [review]
LGTM

**Comment by joshgav** [comment]
Fixed up apostrophes in contractions and landed in 769713e957432ad19ce5181485c73cf4978d57e3

**Pull request closed without merging**